packages/eslint-plugin/README.md

```diff
@@ -1,6 +1,18 @@
 # @fluentui/eslint-plugin
 
-**Custom ESLint rules for Fluent UI**
+**ESLint configuration and custom rules for Fluent UI**
+
+## Configs
+
+Usage: `{ "extends": ["plugin:@fluentui/<name>"] }` or `{ "extends": ["plugin:@fluentui/eslint-plugin/<name>"] }`
+
+- `react`: For `@fluentui/react` (`office-ui-fabric-react`) and related packages
+  - `react--legacy`: Like `react` but requiring an `I` prefix for interfaces
```

Comment:
I don't understand why this is here.

When we deprecate a rule, shouldn't we just add an exclusion to all the packages and then remove it over time?

Reply:
The biggest reason is that the rule which is different ([`@typescript-eslint/naming-convention`](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md)) is a MASSIVE pain to configure because it covers way too many different things--to the point that I made a helper function getNamingConventionRule to handle the variations. (I *really* don't like this rule, but they've removed all the simpler alternatives...) So the alternative to the method you see here would be to make all the ESLint configs JS files and import the helper function. Having a "legacy" config to extend seemed closer to an idiomatic way to do it as far as I could tell from the ESLint ecosystem (though I could be wrong).

The other reason is that the number of different rules may increase over time (including possibly some rules with complex configuration requirements) if we start moving the main config towards the true converged set of rules we want but don't have time to update everything right away (or don't want to update certainly rules at all for legacy code due to philosophical differences). Having a "legacy" config to extend seemed easier than copying all the configuration everywhere.